webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -603,6 +385,347 @@ private static void addScrambles(PdfWriter docWriter, Document doc, ScrambleRequ
+    }
+
+    private static void addFmcScrambleCutoutSheet(PdfWriter docWriter, Document doc, ScrambleRequest scrambleRequest, String globalTitle, int index) throws DocumentException, IOException {
+        Rectangle pageSize = doc.getPageSize();
+        String scramble = scrambleRequest.scrambles[index];
+        PdfContentByte cb = docWriter.getDirectContent();
+
+        BaseFont bf = BaseFont.createFont(BaseFont.HELVETICA, BaseFont.CP1252, BaseFont.NOT_EMBEDDED);
+        BaseFont bfBold = BaseFont.createFont(BaseFont.HELVETICA_BOLD, BaseFont.CP1252, BaseFont.NOT_EMBEDDED);
+
+        int bottom = 30;
+        int left = 35;
+        int right = (int) (pageSize.getWidth()-left);
+        int top = (int) (pageSize.getHeight()-bottom);
+
+        int height = top - bottom;
+        int width = right - left;
+
+        int fontSize;
+        int padding = 90;
+        int marginBottom = 10;
+        int offsetTop = 27;
+
+        int availableScrambleSpace = width - padding;
+        int scrambleFontSize = 20;
+        float scrambleWidth;
+        do {
+            scrambleFontSize--;
+            scrambleWidth = bf.getWidthPoint(scramble, scrambleFontSize);
+        } while (scrambleWidth > availableScrambleSpace);
+
+        int availableScrambleWidth = (int) (width * .45);
+        int availableScrambleHeight = height - (int) (height * .77) - 90;
+        Dimension dim = scrambleRequest.scrambler.getPreferredSize(availableScrambleWidth - 8, availableScrambleHeight - 8);
+        PdfTemplate tp = cb.createTemplate(dim.width, dim.height);
+        Graphics2D g2 = new PdfGraphics2D(tp, dim.width, dim.height, new DefaultFontMapper());
+
+        try {
+            Svg svg = scrambleRequest.scrambler.drawScramble(scramble, scrambleRequest.colorScheme);
+            drawSvgToGraphics2D(svg, g2, dim);
+        } catch (InvalidScrambleException e) {
+            l.log(Level.INFO, "", e);
+        } finally {
+            g2.dispose();
+        }
+
+        cb.setLineDash(3f, 3f);
+        cb.moveTo(left, top - offsetTop + (marginBottom * 3));
+        cb.lineTo(right, top - offsetTop + (marginBottom * 3));
+        cb.stroke();
+
+        final int scramblesPerSheet = 8;
+        for (int y = 0; y < scramblesPerSheet; y++) {
+            cb.beginText();
+            fontSize = 12;
+            cb.setFontAndSize(bfBold, fontSize);
+            if(scrambleRequest.scrambles.length > 1) {
+                cb.showTextAligned(PdfContentByte.ALIGN_LEFT, globalTitle + " - " + scrambleRequest.title + " - Scramble " + (index + 1) + " of " + scrambleRequest.scrambles.length + ":", left, top - offsetTop, 0);
+            } else {
+                cb.showTextAligned(PdfContentByte.ALIGN_LEFT, globalTitle + " - " + scrambleRequest.title + ":", left, top - offsetTop, 0);
+            }
+            offsetTop += fontSize + marginBottom;
+            cb.endText();
+
+            cb.beginText();
+            fontSize = 15;
```

Comment:
This looks strange. Why set fontSize to 15, but then call `cb.setFontAndSize` with `scrambleFontSize`?